package.json

```diff
@@ -17,7 +17,7 @@
   ],
   "scripts": {
     "start": "./bin/server.js",
-    "test": "nodeunit"
+    "test": "jshint bin/ lib/ test/ *.js && nodeunit"
```

Comment:
Should the `exceptions/` directory also be linted? I know it's just some arrays but there are still simple mistakes we can make there.